lib/navigation/preprocess_encoders.cpp

```diff
@@ -50,7 +142,18 @@ void EncodersPreprocessor::checkReliable(const core::EncoderData &encoder_data)
 
   -also figure out return type/ what we update and update
   documentation as appropriate
-  */
+  
+
 }
+*/
+
+void EncodersPreprocessor::checkReliable(const core::EncoderData num_outliers_per_encoder){
```

Comment:
Also include check for if number of unreliable encoders > 1 and add a //TODOLater comment to write a fail state. The high level design of this class will be changing marginally to better represent our ability to fail when necessary but this can be implemented slightly later. For now, have a look at #39, its use of templates and some of the comments I left there to try and remove some of the boilerplate code you have repeated here (it's not technically "wrong" but can be more succinct). If you have any questions on what's happening here, you're always welcome to shoot me a message or ask at meetings.